.github/workflows/coding-standards.yml

```diff
@@ -13,3 +13,5 @@ jobs:
   coding-standards:
     name: "Coding Standards"
     uses: "doctrine/.github/.github/workflows/coding-standards.yml@1.4.1"
+    with:
+      php-version: 8.1
```

Comment:
You can instead bump to `coding-standards.yml@2.0.0`. That version defaults to PHP 8.1.